reactDashboard/src/components/DataCharts/DataCharts.css

```diff
@@ -21,19 +63,112 @@
 
 .sonification-play-button {
   position: absolute;
-  bottom: 10px;
-  left: 10px;
+  top: 20px;
+  right: 40px;
+  background-color: #ffffff;
+  font-size: 0.5rem;
+  border: 1px solid #6c6c6c;
+  border-radius: 10px;
+  padding: 7px;
+  width: 8em;
+}
+
+.sonification-play-button:hover {
+  box-shadow: 5px 5px 5px #aaaaaa;
+}
+
+.sonification-play-button.col {
   border: none;
+  box-shadow: none;
+  display: hidden;
   background: none;
 }
 
+.sonification-play-button h5 {
+  font-size: 0.75rem;
+  color: #6c6c6c;
+  margin-bottom: 0;
+}
+
+#date-range-slider-position {
+  position: absolute;
+  bottom: 0 !important;
+  max-width: 85vw;
+  right: 40px;
+  left: 40px;
+  padding: 0 0 70px;
+  margin: 0 auto;
+}
+
+@media screen and (max-width: 1400px) {
+  #date-range-slider-position {
+    max-width: 80vw;
+  }
+}
+
+@media screen and (max-width: 1200px) {
+  #date-range-slider-position {
+    max-width: 70vw;
+  }
+}
+
+@media screen and (max-width: 993px) {
+  #date-range-slider-position {
+    max-width: 60vw;
+  }
+}
+
+@media screen and (max-width: 750px) {
+  #date-range-slider-position {
+    max-width: 50vw;
+  }
+}
+
+@media screen and (max-width: 995px) {
+  .sonification-play-button {
+    top: 10px;
+    right: 20px;
+  }
+}
+
+@media screen and (max-width: 450px) {
+  .sonification-play-button {
+    font-size: 0.4rem;
+    padding: 10px 5px;
+    top: 20px;
+  }
+  .sonification-play-button h5 {
+    font-size: 0.5rem;
```

Comment:
I don't think reducing the font size here has much benefit

Reply:
removed